docs/content/latest/integrations/jdbc-driver.md

```diff
@@ -0,0 +1,278 @@
+
+  ```sh
+  java -jar yb-sample-apps.jar  \
+       --workload SqlInserts \
+       --num_threads_read 15 --num_threads_write 15 \
+       --nodes 127.0.0.1:5433,127.0.0.2:5433,127.0.0.3:5433
+  ```
+
+The application creates 30 connections, 1 for each reader and writer threads. To verify the behavior, wait for the app to create connections and then visit `http://<host>:13000/rpcz` from your browser for each node to see that the connections are equally distributed among the nodes. 
+This URL presents a list of connections where each element of the list has some information about the connection as shown in the following screenshot. You can count the number of connections from that list, or simply search for the occurrence count of the `host` keyword on that webpage. Each node should have 10 connections.
+
+![Load balancing with host connections](/images/develop/ecosystem-integrations/jdbc-load-balancing.png)
+
+### Check Topology-aware load balancing using yb-sample-apps
+
+For topology-aware load balancing, run the SqlInserts workload application with the `topology-keys1` property set to `aws.us-west.us-west-2a`; only two nodes will be used in this case.  
+
+  ```sh
+  java -jar yb-sample-apps.jar \
+        --workload SqlInserts \
+        --nodes 127.0.0.1:5433,127.0.0.2:5433,127.0.0.3:5433 \
+        --num_threads_read 15 --num_threads_write 15 \
+        --topology_keys aws.us-west.us-west-2a
+  ```
+
+To verify the behavior, wait for the app to create connections and then navigate to `http://<host>:13000/rpcz`. The first two nodes should have 15 connections each, and the third node should have zero connections.
+
+## Clean up
+
+When you're done experimenting, run the following command to destroy the local cluster:
+
+```sh
+./bin/yb-ctl destroy
+```
+
+## Other examples
+
+To access sample applications that use the Yugabyte JDBC driver, visit [YugabyteDB JDBC driver](https://github.com/yugabyte/pgjdbc).
+
+To use the samples, complete the following steps: 
+
+- Install YugabyteDB by following instructions provided in [Quick Start Guide](/latest/quick-start/install/). 
+
+- Build the examples by running `mvn package`.
+
+- Run the `run.sh` script, as per the following guideline:
+
+  ```sh
+  ./run.sh [-v] [-i] -D -<path_to_yugabyte_installation>
+  ```
+
+  In the preceding command, replace:
+
+  - *[-v] [-i]* with `-v` if you want to run the script in `VERBOSE` mode.
+
+  - *[-v] [-i]* with `-i` if you want to run the script in `INTERACTIVE` mode.
+
+  - *[-v] [-i]* with `-v -i` if you want to run the script in both `VERBOSE` and `INTERACTIVE` mode at the same time.
+
+  - *<path_to_yugabyte_installation>* with the path to the directory where you installed YugabyteDB. 
+
+  The following is an example of a shell command that runs the script:
+
+  ```sh
+  ./run.sh -v -i -D ~/yugabyte-2.7.2.0/
```

Comment:
May be we can just a comment here again that the verion could be 2.7.2.0 or higher